tools/RAiDER/aria/calcGUNW.py

```diff
@@ -1,7 +1,102 @@
-def main(args):
-    """ Calculate  interferometric phase delay at dates in ARIA standard product (GUNW) """
-    pass
+"""
+Calculate the interferometric phase from the 4 delays files of a GUNW
+Write it to disk
+"""
+import os
+import xarray as xr
+import numpy as np
+from RAiDER.utilFcns import rio_open
+from RAiDER.logger import logger
+from datetime import datetime
+import h5py
+from netCDF4 import Dataset
 
+## ToDo:
+    # Check difference direction
+
+TROPO_GROUP = 'science/grids/corrections/external/troposphere'
+TROPO_NAMES = ['troposphereWet', 'troposphereHydrostatic']
+
+
+def compute_delays(dct_delays:dict, wavelength):
```

Comment:
The input of this function makes it hard for me to understand.

Easier to read if you expand what you actually need for this function in case it is ever broken apart.

`dct_delays = <Output somewhere>` else and then you explicitly pass what you are using here.

`comput_delays(slc_filename, slc_filename, ...`

That way it's clear to read (and it looks like you only need one.